jekyll.yml

```
@@ -216,6 +216,7 @@ company_name: Kong Inc.
 mesh_product_name: Kong Mesh
 deck_product_name: decK
 contributing_product_name: Contribution Guidelines
+service_catalog_product_name: Service Catalog
```

```suggestion
service_catalog_name: Service Catalog
```
The product key in there feels too long. We don't really have a naming convention for these (though we should). Otherwise looks great!